beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -128,6 +128,10 @@ const MAX_GOSSIP_PROPOSER_SLASHING_QUEUE_LEN: usize = 4_096;
 /// before we start dropping them.
 const MAX_GOSSIP_ATTESTER_SLASHING_QUEUE_LEN: usize = 4_096;
 
+/// The maximum number of queued `LightClientFinalityUpdate` objects received on gossip that will be stored
+/// before we start dropping them.
+const MAX_GOSSIP_FINALITY_UPDATE_QUEUE_LEN: usize = 4_096;
```

Comment:
We should be receiving one finality update per slot right? In that case, it would be nice to keep this value same as the `MAX_SYNC_CONTRIBUTION_QUEUE_LEN` as it has similar frequency of messages. Please correct me if i'm wrong.